src/main/java/com/snowplowanalytics/snowplow/tracker/Tracker.java

```diff
@@ -356,4 +356,14 @@ private void addSubject(Event event, TrackerPayload payload) {
         }
     }
 
+    /**
+     * Attempts to send all remaining events, then shuts down the Emitter so that no more events can be sent.
+     * This method calls the Emitter.close() method. For the default BatchEmitter,
+     * this stops the ScheduledExecutorService and the thread pool (non-daemon threads).
+     * There is no way to restart the Emitter after calling close().
+     */
+    public void close() {
```

Comment:
Nice API comment! Should we also implement the [`java.io.Closeable` interface](https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html) in the `Tracker` class? The `BatchEmitter` implements it and seems like a nice convention to indicate that close should be called.

Reply:
Not sure about this. The Tracker doesn't change when close() is called - it still tracks events just the same as before...

Reply:
That's fair!